### PR TITLE
Setup storybook addons

### PR DESCRIPTION
**Pull request by GoldenVee**
Resolves: Issue #35

## Description
This PR:
- Sets up a11y-addon for storybook
- Reverts storybook version to 6.5

## To Test :mag: 
- Go to Buttons component
- Verify that the accessibility tab is present, should be showing one error for text in the button

## Checklist :white_check_mark:
- [ ] I have looked over the diffs.
- [ ] There are no errors/failing tests from my changes on this branch.
- [ ] I have requested a review on this PR.
- [ ] I have re-assigned the Github ticket and moved the ticket on the project board.
- [ ] If I have added an ENV var or other secrets, I have updated Dashlane and the ReadMe (and any other applicable docs)

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -19,5 +20,7 @@ module.exports = {
   framework: '@storybook/react',
   typescript: {
     reactDocgen: 'react-docgen-typescript-plugin',
+    skipBabel: true,
```
Comment (amahajan87) [inline .storybook/main.js:23]: what would be the reasoning for this?

Comment (GoldenVee) [reply .storybook/main.js:23]: I think that might be a remnant of when I was trying to troubleshoot autodocs. It's no longer necessary, so I'll remove that.

Comment (GoldenVee) [reply .storybook/main.js]: I've removed that line of code, should be good to go now.

**Review by amahajan87 — APPROVED** [review]
LGTM 🎉 Able to test and view Accessibility tab on storybook directly

**Pull request merged**